adapters/nirantk/bert-base-multilingual-uncased-hinglish-sentiment.yaml

```diff
@@ -0,0 +1,70 @@
+# Adapter-Hub adapter entry
+# Defines a single adapter entry in Adapter-Hub
+# --------------------
+
+# The name of the author(s) of this adapter.
+author: "Meghana Bhange, Nirant K"
+
+# A bibtex citation of the work related to this adapter.
+citation: |
+  @article{Hinglish,
+      title={HinglishNLP: Fine-tuned Language Models for Hinglish Sentiment Detection},
+      author={"Meghana Bhange,
+              Nirant Kasliwal,
+      journal={ArXiv},
+      year={2020}
+  }
+
+# The string identifier of the adapter architecture (must be available in architecture).
+# Describes the adapter architecture used by this adapter
+config: # TODO: REQUIRED
+  # The name of the adapter config used by this adapter (a short name available in the `architectures` folder).
+  # Example: pfeiffer
+  using: pfeiffer
+
+# The version to be downloaded if no version is explicitly stated.
+default_version: "1"
+
+# A short description of this adapter.
+description: "Adapter for Hinglish Sentiment Analysis, based on SemEval 2020 Task 9"
+
+# A contact email of the author(s).
+email: "nirant.kasliwal@gmail.com"
+
+# A list of different versions of this adapter available for download.
+files:
+  - sha1: "4e058d2bd9dbcbe204eb04c15e201900ba5dd4c9"
+    # Download URL pointing to a zip folder containing the adapter module.
+    url: "https://public.ukp.informatik.tu-darmstadt.de/AdapterHub/contrib/nirantk/hinglish-bert-base-multilingual-uncased-20201019T062947Z-001.zip"
+    version: "1"
+    description: ""
+
+
+# A GitHub handle associated with the author(s).
+github: "NirantK"
+
+# The hidden size of the model
+hidden_size: 768
+
+# The string identifier of the pre-trained model (by which it is identified at Huggingface).
+# Example: bert-base-uncased
+model_name: "bert-base-multilingual-uncased"
+
+# The model type.
+# Example: bert
+model_type: bert
+
+# The string identifier of the subtask this adapter belongs to.
+subtask: Hinglish
```

Comment:
Not really required, but since you added a new subtask/ dataset, it might be useful to [add some description](https://github.com/Adapter-Hub/Hub/blob/master/TEMPLATES/subtask.template.yaml) for this subtask to improve the presentation on our website.

Reply:
Please see 3abd15d. If this works, please Resolve and Merge?

Reply:
Sorry if that was a bit unclear: What I meant is a separate file describing the dataset you used (see [here](https://github.com/Adapter-Hub/Hub/tree/master/subtasks/text_task)). The description in that file fills the box on the [explore page](https://adapterhub.ml/explore/) of the website and helps users to find the adapter. You can add that file to this PR. Thanks!

Reply:
Hey @calpt -- I've added the file with information on dataset used as well now. Sorry for not getting you yesterday.

All the checks clear as well. Please let me know if I missed anything?

Reply:
Thanks for adding! Just one last thing: the values under the `subtask` attribute in both files must match for the linking to work. Looks good otherwise.

Reply:
Done. Maybe we should add that as check as well so that it doesn't require your time/review in future?

Reply:
Thanks! Yes, adding a check that verifies if the subtask given in an adapter file is available in the subtasks directory is something worth considering.